Makefile

```diff
@@ -25,7 +25,7 @@ DEIS_REGISTRY ?= ${DEV_REGISTRY}/
 # Kubernetes-specific information for RC, Service, and Image.
 RC := manifests/deis-${SHORT_NAME}-rc.yaml
 SVC := manifests/deis-${SHORT_NAME}-service.yaml
-IMAGE := ${DEIS_REGISTRY}${IMAGE_PREFIX}/${SHORT_NAME}:${VERSION}
+IMAGE := ${DEIS_REGISTRY}/${SHORT_NAME}:${VERSION}
```

Comment:
That is just for my local testing.I need to rebase this and clean a lot of changes before it becomes mergable